dotCMS/src/main/java/com/dotcms/system/event/local/type/publish/AddedToQueueEvent.java

```diff
@@ -0,0 +1,20 @@
+package com.dotcms.system.event.local.type.publish;
+
+import com.dotcms.publisher.business.PublishQueueElement;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Object used to represent an event to be triggered when a new asset is queued to publish
+ *
+ * @author nollymar
+ */
+public class AddedToQueueEvent extends PublishEvent {
+
+    public AddedToQueueEvent(List<PublishQueueElement> publishQueueElements){
+        this.setName(AddedToQueueEvent.class.getCanonicalName());
+        this.setPublishQueueElements(publishQueueElements);
+        this.setDate(new Date());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)